compiler/rustc_ast_passes/src/feature_gate.rs

```diff
@@ -692,7 +692,7 @@ fn check_new_solver_banned_features(sess: &Session, features: &Features) {
     {
         sess.dcx().emit_err(errors::IncompatibleFeatures {
             spans: vec![gce_span],
-            f1: Symbol::intern("-Znext-solver=globally"),
+            f1: sym::dash_z_next_dash_solver_equals_globally,
```

Comment:
This is a bit of a readability loss sadly. Some kind of inline macro to preintern strings would be very cool